src/common/persistence/timescaledb/entities/sum-daily.entity.ts

```diff
@@ -0,0 +1,32 @@
+import { PrimaryColumn, ViewColumn, ViewEntity } from 'typeorm';
+
+@ViewEntity({
+  expression: `
+    SELECT
+      time_bucket('1 day', timestamp) AS time, series, key,
+      last(value, timestamp) AS last,sum(value) AS sum
```

Comment:
You can remove `last(value, timestamp) AS last` from the query. I don't see it being used in the entity

Reply:
I'll do this later on, with the next prs